public/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  <link rel="stylesheet" href="/app.css">
+    <meta charset="utf-8">
+    <title>09-skate-page</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-nav">
+        <button class="button"><span>SUBSCRIBE</span><i class="fa fa-circle"></i></button>
+       <nav class="top-nav-icons">
+        <i class="fa fa-ellipsis-v"></i>
+        <i class="fa fa-facebook-official"></i>
+        <i class="fa fa-rss-square"></i>
+        <i class="fa fa-twitter-square"></i>
+        <i class="fa fa-youtube-square"></i>
+        </nav>
+        </nav>
+        <nav class="red-button">
+          <button><i class="fa fa-minus"></i><span>SKBT NEWS</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>SKATEBOARDERS</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>BOARD DESIGN</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>CONTACT US</span><i class="fa fa-minus"></i></button>
+        </nav>
+
+         <img class="top-img"src="/img/top-image.png" alt=""/>
+       <div class="icon-selector"
+         <i class="fa fa-circle-thin"></i><i class="fa fa-circle-thin"></i><i class="fa fa-circle-thin"></i>
+       </div>
+       <section class="arti">
+         <article class="content">
+           <h3>Tony Hawk Foundation Raises Big Money At Celeb Fundraiser</h3>
+           <h4>October 10, 2013</h4>
+           <p> It was just over 90 degrees in Los Angeles, but that didn't stop the throngs from coming out to celebrate the 10th annual tony Hawks's Stand up for Skateparks Benefit Presented by SLS Hotel & Casino Las Vegas on Saturday at the Green Acres Estate in Beverly Hills.</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+         <article class="content">
+           <h3>Rodney Mullen: Pop an oliie and innovate! (TED Talk)</h3>
+           <h4>March 10, 2013</h4>
+           <p>The last thing Rodney Mullen, the godfather of street skating, wanted were competitive victories. In the exuberant talk he shares his love of teh open skateboarding community and how teh unique environments it plays in drive the creation of new tricks--fostering...</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+         <article class="content">
+           <h3>Ray Barbee Talks 'Ban This'</h3>
+           <h4>October 04, 2013</h4>
+           <p>Since filming is in full swing for the Vans video Adventures With Chris has taken the oppertunity to sit some of the team down to look behind the skating of some of thier earlier parts. Today we sit down Ray Barbee, one of the smoothest guys to ever step on a skateboard and Chris' childhood idol...</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+        </section>
+        <button class="the arrow"><span>SKATEBOARDERS</span></button>
```

Comment:
This should be a heading not a button. This should be grouped with the images so that you can position it.